cmd/drand-cli/cli.go

```diff
@@ -125,6 +125,12 @@ var periodFlag = &cli.StringFlag{
 	Usage: "period to set when doing a setup",
 }
 
+var catchupPeriodFlag = &cli.StringFlag{
```

Comment:
Why not use [`DurationFlag`](https://godoc.org/github.com/urfave/cli#DurationFlag)?

Reply:
consistency with existing `periodFlag`